jobs/cc_deployment_updater/monit

```diff
@@ -4,7 +4,16 @@ check process cc_deployment_updater
   start program "/var/vcap/jobs/bpm/bin/bpm start cc_deployment_updater"
   stop program "/var/vcap/jobs/bpm/bin/bpm stop cc_deployment_updater"
   group vcap
+  <% if_p("cc.thresholds.api.alert_if_above_memory_percent") do %>
+  if totalmem > <%= p("cc.thresholds.api.alert_if_above_memory_percent") %> % for 3 cycles then alert
+  <% end.else do %>
   if totalmem > <%= p("cc.thresholds.api.alert_if_above_mb") %> Mb for 3 cycles then alert
+  <% end %>
+
+  <% if_p("cc.thresholds.api.restart_if_consistently_above_memory_percent") do %>
+  if totalmem > <%= p("cc.thresholds.api.restart_if_consistently_above_memory_percent") %> % for 15 cycles then restart
```

Comment:
**question**: Will this restart drain web workers?

Reply:
Well this line specifically is for the cc_deployment_updater. The existing behaviour is to just issue a monit restart.

The cloud_controller_ng monit template peforms a custom drain